### PR TITLE
DVX-59 added linter config to webapp & linted files 

### PR DESCRIPTION
**Pull request by jean-noumena**

### CODE REVIEW

File: webapp/.eslintrc.cjs [hunk 1]
```diff
@@ -10,9 +10,7 @@ module.exports = {
     parser: '@typescript-eslint/parser',
     plugins: ['react-refresh'],
     rules: {
```
Comment (lukasz-noumena) [inline webapp/.eslintrc.cjs:12]: You may think about quotes 
```
quotes: [
      'error',
      'single',
      { avoidEscape: true, allowTemplateLiterals: true },
    ]
```

**Review by lukasz-noumena — DISMISSED** [review]
Looks good to me. But maybe others will contribute with some extra config

File: webapp/src/services/BaseService.ts [hunk 2]
```diff
@@ -30,15 +30,16 @@ export class BaseService {
                 this.withAuthorizationHeader()
             )
 
-            source.onmessage = function (_: MessageEvent) {
+            // _: event-source-polyfill MessageEvent
+            source.onmessage = function () {
```
Comment (lukas-noumena) [inline webapp/src/services/BaseService.ts:34]: convert to arrow function for consistency sake

File: webapp/src/services/BaseService.ts [hunk 3]
```diff
@@ -49,7 +50,7 @@ export class BaseService {
                 source.close()
```
Comment (lukas-noumena) [inline webapp/src/services/BaseService.ts:50]: might want to check if this also removes the eventlistener, to avoid potential memory leaks

**Review by lukas-noumena — APPROVED** [review]

**Review by lukas-noumena — APPROVED** [review]

**Pull request merged**